vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/components/view/TouchEventEmitter.cpp

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "TouchEventEmitter.h"
+
+namespace facebook::react {
+
+#pragma mark - Touches
+
+static void setTouchPayloadOnObject(
```

Comment:
As long as you're modifying this file for now, you'll probably also want to add a "button", "ctrlKey", "shiftKey", etc. property to Touch.h to get right click onPress suppression working via Pressability :)